### PR TITLE
Add pubdate for torrent providers that has universal time format

### PR DESCRIPTION
**Pull request by fernandog**
@duramato

### CODE REVIEW

**Comment by fernandog** [comment]
@duramato need you to test Transmistthenet because I don't have it.

File: medusa/providers/nzb/anizb.py [hunk 1]
```diff
@@ -115,7 +114,7 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
             return results
 
     def _get_size(self, item):
-        """Override the default _get_size to prevent it from extracting using it the default tags"""
+        """Override the default _get_size to prevent it from extracting using it the default tags."""
```
Comment (fernandog) [inline medusa/providers/nzb/anizb.py]: so what is the correct sentence?

File: medusa/providers/torrent/rss/nyaatorrents.py [hunk 2]
```diff
@@ -26,11 +26,11 @@
...
 
-class NyaaProvider(TorrentProvider):  # pylint: disable=too-many-instance-attributes
+class NyaaProvider(TorrentProvider):
+    """Nyaa Torrent provider."""
 
     def __init__(self):
-
-        # Provider Init
+        """Initialize the class."""
         TorrentProvider.__init__(self, 'NyaaTorrents')
```
Comment (ratoaq2) [inline medusa/providers/torrent/rss/nyaatorrents.py]: All the constructors should use super call instead of this. 😉

Comment (fernandog) [inline medusa/providers/torrent/rss/nyaatorrents.py]: no ideia how to do it

**Comment by fernandog** [comment]
@ratoaq2 help?

`/home/travis/build/pymedusa/Medusa/medusa/providers/nzb/newznab.py:32:1: F811 redefinition of unused 'app' from line 28`

File: medusa/providers/generic_provider.py [hunk 3]
```diff
@@ -358,75 +364,91 @@ def find_search_results(self, show, episodes, search_mode, forced_search=False,
...
         kwargs['hooks'] = {'response': self.get_url_hook}
         return getURL(url, post_data, params, self.headers, timeout, self.session, **kwargs)
 
     def image_name(self):
+        """Return provider image name."""
         return self.get_id() + '.png'
 
-    def is_active(self):  # pylint: disable=no-self-use
+    def is_active(self):
+        """Check is provider is active."""
```
Comment (medariox) [inline medusa/providers/generic_provider.py]: """Check **is** provider is active."""
Should be if

**Comment by fernandog** [comment]
fixed all reviews requets

File: medusa/providers/torrent/html/limetorrents.py [hunk 4]
```diff
@@ -15,14 +15,15 @@
...
+"""Provider code for Limetorrents."""
 from __future__ import unicode_literals
 
 import re
 import traceback
-
 from contextlib2 import suppress
 import requests
+
+# from dateutil import parser
```
Comment (ratoaq2) [inline medusa/providers/torrent/html/limetorrents.py]: delete this line and the blank line

**Comment by fernandog** [comment]
any more comments? just waiting approve

**Pull request merged**